snap/snapcraft.yaml

```diff
@@ -1,7 +1,7 @@
 name: eclipse
-base: core22
+base: core24
```

Comment:
Just to confirm, have you tested this works as expected? I've not seen many desktop apps on core24 yet?

Reply:
Okay, got it working on a fresh box, let me merge and see if the automated screenshotting all works!

Reply:
Sorry, I can't test locally due to https://bugs.launchpad.net/snapcraft/+bug/2069292